Utilities/VisualStudioProjectTemplates/EdFi.ProjectTemplates.Extensions/Properties/AssemblyInfo.cs

```diff
@@ -14,7 +14,7 @@
 [assembly: AssemblyConfiguration("")]
 [assembly: AssemblyCompany("")]
 [assembly: AssemblyProduct("Ed-Fi API Extensions Project Template")]
-[assembly: AssemblyCopyright("Copyright © Ed-Fi Alliance 2018")]
+[assembly: AssemblyCopyright("Copyright © Ed-Fi Alliance " + DateTime.now().year)]
```

Comment:
and Contributors is missing here .  And also check required wordings "Copyright  © $(date +'%Y') Ed-Fi Alliance, LLC and Contributors"